src/ReactiveUI/Activation.cs

```diff
@@ -54,6 +54,9 @@ public sealed class ViewModelActivator
         /// <value>The deactivated.</value>
         public IObservable<Unit> Deactivated { get { return deactivated; } }
 
+        /// <summary>
```

Comment:
In terms of compiler warnings (can't remember exactly what you get for missing a returns...), would you prefer empty comments or nothing at all?